webui/react/src/components/ManageGroupsModal.tsx

```diff
@@ -0,0 +1,102 @@
+import { Select } from 'antd';
+import React, { useEffect } from 'react';
+
+import Form from 'components/kit/Form';
+import { Modal } from 'components/kit/Modal';
+import useFeature from 'hooks/useFeature';
+import { updateGroup } from 'services/api';
+import { V1GroupSearchResult } from 'services/api-ts-sdk';
+import Spinner from 'shared/components/Spinner';
+import { ErrorType } from 'shared/utils/error';
+import { DetailedUser } from 'types';
+import { message } from 'utils/dialogApi';
+import handleError from 'utils/error';
+
+const FIELD_NAME = 'groups';
+
+interface Props {
+  groups: V1GroupSearchResult[];
+  user: DetailedUser;
+}
+
+const ManageGroupsModalComponent: React.FC<Props> = ({ user, groups }: Props) => {
+  const [form] = Form.useForm();
```

Comment:
good to add form values type like Form.useForm<FORM_TYPE>();
[example](https://github.com/determined-ai/determined/blob/master/webui/react/src/components/ModelCreateModal.tsx#L50)